STYLE.md

```diff
@@ -105,17 +105,17 @@
 A few general notes on our error handling philosophy:
 
 * All error code returns should be checked.
-* At a very high level, our philosophy is that errors that are *likely* to happen should be
-  gracefully handled. Examples of likely errors include any type of network error, disk IO error,
-  bad data returned by an API call, bad data read from runtime files, etc. Errors that are
-  *unlikely* to happen should lead to process death, under the assumption that the additional burden
-  of defensive coding and testing is not an effective use of time for an error that should not happen
-  given proper system setup. Examples of these types of errors include not being able to open the shared
-  memory region, an invalid initial JSON config read from disk, system calls that should not fail
-  assuming correct parameters (which should be validated via tests), etc. Examples of system calls
-  that should not fail when passed valid parameters include most usages of `setsockopt()`,
-  `getsockopt()`, the kernel returning a valid `sockaddr` after a successful call to `accept()`,
-  `pthread_create()`, `pthread_join()`, etc.
+* At a very high level, our philosophy is that errors caused by untrusted network traffic or
+  that are raised by the Envoy process environment and are *likely* to happen should be gracefully
```

Comment:
This strikes me as a classic example of an english sentence that could use parenthesis :-}.  Could you clarify whether you mean "(caused by untrusted network traffic or that are raised by the Envoy process environment) and are *likely* to happen" or ""caused by untrusted network traffic or (that are raised by the Envoy process environment and are *likely* to happen)"?  It seems to me a key distinction.